.phpunit.result.cache

```diff
@@ -0,0 +1 @@
+C:37:"PHPUnit\Runner\DefaultTestResultCache":2827:{a:2:{s:7:"defects";a:1:{s:94:"LaminasTest\Permissions\Rbac\Assertion\CallbackAssertionTest::testCallbackIsDecoratedAsClosure";i:4;}s:5:"times";a:35:{s:94:"LaminasTest\Permissions\Rbac\Assertion\CallbackAssertionTest::testCallbackIsDecoratedAsClosure";d:0.02;s:96:"LaminasTest\Permissions\Rbac\Assertion\CallbackAssertionTest::testAssertMethodPassRbacToCallback";d:0.002;s:78:"LaminasTest\Permissions\Rbac\Assertion\CallbackAssertionTest::testAssertMethod";d:0.001;s:84:"LaminasTest\Permissions\Rbac\Assertion\CallbackAssertionTest::testAssertWithCallable";d:0;s:88:"LaminasTest\Permissions\Rbac\Assertion\CallbackAssertionTest::testAssertWithInvalidValue";d:0.004;s:61:"LaminasTest\Permissions\Rbac\RbacTest::testIsGrantedAssertion";d:0.003;s:62:"LaminasTest\Permissions\Rbac\RbacTest::testIsGrantedSingleRole";d:0;s:62:"LaminasTest\Permissions\Rbac\RbacTest::testIsGrantedChildRoles";d:0;s:67:"LaminasTest\Permissions\Rbac\RbacTest::testIsGrantedWithInvalidRole";d:0;s:50:"LaminasTest\Permissions\Rbac\RbacTest::testGetRole";d:0;s:50:"LaminasTest\Permissions\Rbac\RbacTest::testHasRole";d:0.002;s:68:"LaminasTest\Permissions\Rbac\RbacTest::testHasRoleWithInvalidElement";d:0;s:59:"LaminasTest\Permissions\Rbac\RbacTest::testGetUndefinedRole";d:0;s:60:"LaminasTest\Permissions\Rbac\RbacTest::testAddRoleFromString";d:0;s:59:"LaminasTest\Permissions\Rbac\RbacTest::testAddRoleFromClass";d:0;s:58:"LaminasTest\Permissions\Rbac\RbacTest::testAddRoleNotValid";d:0;s:70:"LaminasTest\Permissions\Rbac\RbacTest::testAddRoleWithParentsUsingRbac";d:0;s:79:"LaminasTest\Permissions\Rbac\RbacTest::testAddRoleWithAutomaticParentsUsingRbac";d:0;s:61:"LaminasTest\Permissions\Rbac\RbacTest::testAddCustomChildRole";d:0.048;s:64:"LaminasTest\Permissions\Rbac\RbacTest::testAddMultipleParentRole";d:0.001;s:56:"LaminasTest\Permissions\Rbac\RbacTest::testAddParentRole";d:0;s:51:"LaminasTest\Permissions\Rbac\RbacTest::testGetRoles";d:0;s:53:"LaminasTest\Permissions\Rbac\RbacTest::testEmptyRoles";d:0;s:54:"LaminasTest\Permissions\Rbac\RoleTest::testConstructor";d:0;s:50:"LaminasTest\Permissions\Rbac\RoleTest::testGetName";d:0;s:56:"LaminasTest\Permissions\Rbac\RoleTest::testAddPermission";d:0;s:60:"LaminasTest\Permissions\Rbac\RoleTest::testInvalidPermission";d:0;s:51:"LaminasTest\Permissions\Rbac\RoleTest::testAddChild";d:0;s:52:"LaminasTest\Permissions\Rbac\RoleTest::testAddParent";d:0;s:62:"LaminasTest\Permissions\Rbac\RoleTest::testPermissionHierarchy";d:0;s:67:"LaminasTest\Permissions\Rbac\RoleTest::testCircleReferenceWithChild";d:0.002;s:68:"LaminasTest\Permissions\Rbac\RoleTest::testCircleReferenceWithParent";d:0;s:57:"LaminasTest\Permissions\Rbac\RoleTest::testGetPermissions";d:0;s:58:"LaminasTest\Permissions\Rbac\RoleTest::testAddTwoChildRole";d:0;s:56:"LaminasTest\Permissions\Rbac\RoleTest::testAddSameParent";d:0;}}}
```

Comment:
Should be excluded from Repo through .gitignore

Reply:
thanks, I will delete this file and add in .gitignore